lib/sift/client.rb

```diff
@@ -720,6 +720,71 @@ def apply_decision!(configs = {})
       handle_response(apply_decision(configs))
     end
 
+    def build_default_headers_post(api_key)
+      { 
+        "Authorization" => "Basic #{Base64.encode64(api_key+":")}",
+        "User-Agent" => "SiftScience/v#{@version} sift-ruby/#{VERSION}",
+        "Content-Type" => "application/json"
+      }      
+    end
+
+    def send(properties = {}, opts = {})
```

Comment:
the name of the method looks too generic, let's rename it to `verification_send`